src/core/core.scale.js

```diff
@@ -693,8 +695,22 @@ module.exports = function(Chart) {
 						labelY = me.bottom - labelYOffset;
 					}
 
-					var xLineValue = me.getPixelForTick(index) + helpers.aliasPixel(lineWidth); // xvalues for grid lines
-					labelX = me.getPixelForTick(index, gridLines.offsetGridLines) + optionTicks.labelOffset; // x values for optionTicks (need to consider offsetLabel option)
+					var xLineValue; // xvalues for grid lines
+					if (gridLines.offsetGridLines && ticks.length > 1) {
+						if (index === 0) {
+							xLineValue = (me.getPixelForTick(0) * 3 - me.getPixelForTick(1)) / 2;
```

Comment:
When `gridLines.offsetGridLines` is true and a tick is the leftmost tick, grid lines move to the left by one half of the tick interval between itself and the next tick on the right.